WordPress/gutenberg
# Tag Cloud: Add border block support

## What?

Add border block support to the `Tag Cloud` block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?
`Tag Cloud` block is missing border support.

## How?

Adds the border block support in block.json

## Testing Instructions

- Go to Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
- Make sure that `Tag Cloud`  block's border is configurable via Global Styles
- Verify that Global Styles are applied correctly in the editor and frontend
- Edit template/page, Add `Tag Cloud`  block and Apply the border styles
- Verify that block styles take precedence over global styles
- Verify that block borders display correctly in both the editor and frontend
 

## Screenshots or screencast 
In backend:
 
<img width="1210" alt="image" src="https://github.com/user-attachments/assets/8e730929-9ae1-4247-a1f0-1d98754c3d9f">

In frontend:
<img width="762" alt="image" src="https://github.com/user-attachments/assets/91e73eb2-c60e-46a7-89a4-58fc7017fb04">
